site/src/pages/components/selectlist.mdx

```diff
@@ -46,10 +46,10 @@ Because the various parts of the selectlist can be styled, it's important to und
 <SelectAnatomy />
 
 - `<selectlist>` - The root element that contains the button and listbox.
-- `slot=button` - The button element that triggers the visibility of the listbox. If this part is not provided by the author, then selectlist will automatically create one.
-- `slot=selected-value` - The element that displays the value of the currently selected option. If this part is not provided by the author, then selectlist will automatically create one. Note that this part does not necessarily have to be placed inside the `slot=button` part (read on for more information about how to slot your own markup).
-- `slot=marker` - Something that indicates this button opens a listbox, e.g. an icon. If this part is not provided by the author, then selectlist will automatically create one.
-- `slot=listbox` - The wrapper that contains the `<option>`(s) and `<optgroup>`(s). If this part is not provided by the author, then selectlist will automatically create one.
+- button (slot) - The portion of the element which is always visible and in flow with the page, unlike the listbox. It should contain a button to open the listbox. If this part is not provided by the author, then selectlist will automatically create one. All child elements of the selectlist, except `<listbox>`, `<option>`s, and `<optgroup>`s will be slotted into this slot.
```

Comment:
@josepharhar I propose not using the term slot anywhere and this anatomy should not have any slots and thus not have the "toggle".

Reply:
Additionally, I don't get this statement: 

> The portion of the element which is always visible and in flow with the page, unlike the listbox.

It may not be visible and it may not be in flow (eg: I should be able to position this out of flow). I think a more clear definition would be that this will be the element that invokes the listbox as a popover via the popover API or something to that affect

Reply:
> I propose not using the term slot anywhere and this anatomy should not have any slots and thus not have the "toggle".

When I talked to domenic earlier, he didn't sound very opposed to defining the user-agent shadowroot of selectlist in the HTML spec because we already sort of do so for the details element and mention "slots": https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements

For this reason, I feel comfortable going forward using this terminology until we encounter further issues. Is there any other reason you don't want to use the word slot?

> this will be the element that invokes the listbox as a popover

But this is exactly what `<button type=selectmenu>` is. I could get rid of the "button (slot)" section entirely, but I think its worth spelling out since we are explicitly choosing to have a separate button slot and button type=selectmenu in order to support split buttons

> It may not be visible and it may not be in flow (eg: I should be able to position this out of flow).

Good point, I rewrote the first sentence

Reply:
+1 we should mention how this works via slots. @domenic ’s point was about the developer side of this - that you shouldn’t need to use the `slot` attribute. But I think it’s important (for interop, spec ease, and stylability) that we define the structure of the shadow root, and define the algorithm used to slot in content.